_data/meltano/utilities/tableau/gabilew.yml

```
@@ -0,0 +1,56 @@
+commands:
+  refresh:
+    args: refresh
+    description: Refresh a Tableau datasource.
+definition: |
+  Tableau is a widely used data visualisation and exploration tool.
```

```suggestion
  is used to trigger a refresh of a Tableau data source once the data has been updated. Usually this is run in a pipeline following a transformation (i.e. dbt) step that altered the data that Tableau is connected to.
```

@janhoon this looks great, we can probably go ahead and merge it in at this point. Adding a small description like this will be helpful for explaining what the extension is attempting to do with Tableau. Feel free to re-word it or add more context if I got anything wrong.

---

Very happy with the recommendation, I do see the linter failed, will commit the suggestion and take a look at what happened

---

Linter issue resolved itself